lib/sync/index.js

```diff
@@ -132,6 +135,7 @@ function start(cb) {
     },
     function startSyncScheduler(callback) {
       syncScheduler = new SyncScheduler();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "var" keyword to this declaration of "syncScheduler". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2703)